### ISSUE TITLE
The div with class="menu-bar_menu-bar-item_oLDa- menu-bar_hoverable_c6WFB" for the header on the create page has an aria-label attribute but does not have a role attribute. The aria-label attribute is not welll supported on a div with no role attribute.

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7183]
### Where To Find Bug
create

### Expected Behaviour
The screen reader should announce the value of the arial-label attribute.

### Actual Behaviour
In this case, the screen reader does announce the value of the aria-label attribute. However, aria standards demand that a role attribute be used alongside the aria-label attribute.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Tab through the navigation bar in using interactive mode
- Notice that the screen reader does announce the aria-label value, but the aria role attribute should be used with it.

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
